docs/workflow/README.md

```diff
@@ -16,13 +16,73 @@ The runtime repo can be built from a regular, non-admin command prompt. The repo
 
 For information about the different options available, supply the argument `-help|-h` when invoking the build script:
 ```
-libraries -h
+build -h
 ```
 On Unix, arguments can be passed in with a single `-` or double hyphen `--`.
 
+### Configurations
+
+Sometimes you want to build this repo in a mixture of configurations. First, a quick reminder of some concepts -- see the [glossary](../project/glossary.md) for more on these:
```

Comment:
> Sometimes you want to build this repo in a mixture of configurations

Is `Sometimes` the appropriate word here? For someone working on the libraries, I would imagine `Often` is more appropriate. Saying `Sometimes` makes this sound like it isn't common, when really it is super common.

Maybe saying "If you are working under src/libraries, you want to build this repo in a mixture of configurations."